synapse/models/infotech.py

```diff
@@ -329,9 +329,18 @@ def getModelDefs(self):
                     ('latlong', ('geo:latlong', {}), {
                         'doc': 'The last known location for the host.'
                     }),
-
                     ('os', ('it:prod:softver', {}), {
-                        'doc': 'The operating system of the host.'}),
+                        'doc': 'The operating system of the host.'
+                    }),
+                    ('manu', ('str', {}), {
+                        'doc': 'The manufacturer.',
```

Comment:
We've been trying to make these "complete sentences".  Probably want "The manufacturer of the host."

Reply:
Neither is a complete sentence